about.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta
+  name="description"
+  content="This is a website about Stanmega's Web Development Courses."
+/>
+<title>Stanmega-Tech Web Development Courses</title>
+<link rel="stylesheet" href="./about.css" type="text/css" />
+<script
+  src="https://kit.fontawesome.com/e1fe8e6842.js"
+  crossorigin="anonymous"
+></script>
+<script src="menu-mobile.js" defer></script>
+<script src="featured-profs.js" defer></script>
+</head>
+<body>
+  <!--Navigation Bar Design-->
+  <nav>
+    <div class="topmost-bar">
+      <span><i class="fas fa-times fa-2x" id="exit-button"></i></span>
+      <ul>
+         <li><i class="fa fa-facebook"></i></li>
+         <li><i class="fab fa-twitter"></i></li>
+         <li><a href="#">English</a></li>
+         <li><a href="#">My Page</a></li>
+         <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <div class="top-navbar">
+      <div class="logo">
+        <i class="fa fa-university" aria-hidden="true">
```

Comment:
- [ ] Your logo here is looking smaller than the one on the homepage, so it would be nice if you can make them the same size, so please work on this issue.